### PR TITLE
Ca certificate tests

### PR DESCRIPTION
**Pull request by abolla22**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,18 +15,13 @@ source:
 
 build:
   number: 2
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:17]: We should bump build number to `3`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,18 +15,13 @@ source:
...
-    - /usr/bin/curl --cacert %LIBRARY_PREFIX%\\ssl\\cacert.pem https://www.google.com  # [win]
+test:
+  requires:
+    - python <3.10
+    - pip
+  commands:
+    - pip check
 
 about:
   home: https://github.com/conda-forge/ca-certificates-feedstock
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:27]: license should be `MPL-2.0`

Comment (skupr-anaconda) [reply recipe/meta.yaml:27]: - No documentation url provided
- No development url provided

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/run_test.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+done
+user_cert(){
+exit_status=$1
+if [ $exit_status -eq 0 ]; then
+  echo "Able to use ca certificate"
+else
+  echo "Failed to use ca cert files"
+  exit 1
+fi
+}
```
Comment (remkade) [inline recipe/run_test.sh:25]: Can you fix the indentation here?

**Review by remkade — CHANGES_REQUESTED** [review]
Looks good! I like the `meta.yaml` cleanup.

File: recipe/run_test.sh (hunk 3 above)
Comment (abolla22) [reply recipe/run_test.sh]: Done

**Review by skupr-anaconda — APPROVED** [review]

**Comment by abolla22** [comment]
@remkade  indentation has been fixed can u please review the PR when you get a chance.

**Review by remkade — APPROVED** [review]
Looks good!

**Pull request merged**